pkg/runner/executable_path_resolving_runner_test.go

```diff
@@ -0,0 +1,147 @@
+package runner_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/buildbarn/bb-remote-execution/internal/mock"
+	runner_pb "github.com/buildbarn/bb-remote-execution/pkg/proto/runner"
+	"github.com/buildbarn/bb-remote-execution/pkg/runner"
+	"github.com/buildbarn/bb-storage/pkg/filesystem"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
```

Comment:
Please separate imports by second level domain name: just add a newline between all the github.com and golang.org.